### PR TITLE
Load lago.conf defaults when no config file exists

### PR DESCRIPTION
**Pull request by nvgoldin**
fixes https://github.com/lago-project/lago/issues/310

### CODE REVIEW

**Comment by nvgoldin** [comment]
ci test please

File: tests/unit/test_config.py [hunk 1]
```diff
@@ -123,3 +124,23 @@ def test_user_shadows_system():
...
     assert config.get('system_var_2') == 'system_val_2'
+
+
+@mock.patch('os.environ', {})
+@mock.patch.dict('lago.config._cache', {})
+@mock.patch('lago.config._SYSTEM_CONFIG_DIR', '')
+@mock.patch('lago.config._USER_CONFIG', '')
+def test_get_only_defaults():
+    result = {k: config.get(k) for k in config.DEFAULTS.get('lago').keys()}
+    assert (
```
Comment (pilou-) [inline tests/unit/test_config.py]: Wouldn't it be sufficient to use `assert config.DEFAULTS.get('lago') == result` ?

File: lago/config.py [hunk 2]
```diff
@@ -17,27 +17,30 @@
 #
 # Refer to the README and COPYING files for full details of the license
 #
-import ConfigParser
```
Comment (nvgoldin) [inline lago/config.py]: good catch, thanks!

File: tests/unit/test_config.py (hunk 1 above)
Comment (nvgoldin) [inline tests/unit/test_config.py]: True. had some false ordering issue in my mind, will re factor this, thanks.

**Comment by nvgoldin** [comment]
Not needed once https://github.com/lago-project/lago/pull/324 is merged.

**Pull request closed without merging**